Abracadabra/abracadabra_web_app/src/components/QuestionBody.js

```diff
@@ -1,18 +1,23 @@
 import React, { useState, useEffect } from "react";
 import { Row, Col } from "react-bootstrap";
 import { slice } from "__mocks__/fileMock";
+import QuestionService from 'src/services/QuestionService';
 
 function Title({ question, subject, search, searchLength }) {
   const [message, setMessage] = useState("");
   const [questions, setQuestions] = useState([]);
+  const [tagexists, setTagExists] = useState();
```

Comment:
Maybe add a default bool state either true or false to tagexists(if its a bool)

Reply:
> Maybe add a default bool state either true or false to tagexists(if its a bool)

ye true